src/theory/datatypes/datatypes_rewriter.cpp

```diff
@@ -105,6 +108,52 @@ RewriteResponse DatatypesRewriter::postRewrite(TNode in)
       return RewriteResponse(REWRITE_AGAIN_FULL, res);
     }
   }
+  else if (k == kind::DT_SYGUS_EVAL)
```

Comment:
you are in ``kind`` namespace, remove the prefix (the same for the several occurrences in the changes below)